integration/kubernetes/helm-chart/monitor/values.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# The fullnameOverride should not be modified in the usual case.
+fullnameOverride: alluxio-monitor
+imagePullPolicy: IfNotPresent
+
+# The grafana plugin path config, include datasource path and dashboards path.
+grafanaConfig:
+- name: grafana-dashboard-config
+  path: /etc/grafana/provisioning/dashboards
+- name: grafana-datasource-config
+  path: /etc/grafana/provisioning/datasources
+
+# The prometheus.yml file path.
+prometheusConfig:
+  - name: prometheus-config
+    path: /etc/prometheus
+
+## prometheus ##
+
+prometheus:
+  enabled: true
+  imageInfo:
+    image: prom/prometheus
+    imageTag: latest
+  port:
+    TCP: 9090
+  args:
+    - "--config.file=/etc/prometheus/prometheus.yml"
+    - "--storage.tsdb.path=/prometheus" # Where prometheus writes its database.
+    - "--storage.tsdb.retention=72h" # When to remove old data
+    - "--web.listen-address=:9090" # Listen address
+  hostNetwork: true
+  dnsPolicy: ClusterFirstWithHostNet
```

Comment:
users may prefer default to not using hostNetwork, because that's a more common permission in real clusters.

Reply:
Done